panedr/panedr.py

```diff
@@ -421,11 +421,28 @@ def edr_to_df(path, verbose=False):
             all_energies.append([frame.t] + [ener.e for ener in frame.ener])
 
     end = time.time()
-    if verbose:
+    if verbose_set:
         print('\rLast Frame read : {}, time : {} ps'
               .format(ifr, frame.t),
               end='', file=sys.stderr)
         print('\n{} frame read in {:.2f} seconds'.format(ifr, end - begin),
               file=sys.stderr)
+
+    return all_energies, all_names, times
+
+
+def edr_to_df(path: str, verbose: bool = False):
+    import pandas
+    all_energies, all_names, times = read_edr(path, verbose_set=verbose)
     df = pandas.DataFrame(all_energies, columns=all_names, index=times)
     return df
+
+
+def edr_to_dict(path: str, verbose: bool = False):
+    import numpy as np
```

Comment:
Is this an optional dependency? I think its probably safe to make `numpy` compulsory for panedr.  If we do make it compulsory you can import this at the top of the file otherwise use a guard like the one for pandas mentioned in my other comment. 

Thoughts @jbarnoud?

Reply:
I think it is reasonable to make numpy a compulsory dependency. The current users have to install it already because of pandas anyway; the new users will likely use it as well.